packages/core/src/Stores/chart-barrier-store.js

```diff
@@ -87,10 +86,6 @@ export class ChartBarrierStore {
         this.onChartBarrierChange(...barriersToString(this.relative, high, low));
     }
 
-    updateBarrierColor(is_dark_mode) {
-        this.color = is_dark_mode ? BARRIER_COLORS.DARK_GRAY : BARRIER_COLORS.GRAY;
```

Comment:
@akmal-deriv I got a question: Currently, this function is normally called when there is an ongoing contract with a barrier on the main Trading page or in Contract details.
What's the reason for removing this logic? What are we replacing it with? :)
Is it correct that we won't be updating the barrier color anymore via deriv-app, and it's gonna be done internally inside SmartCharts? I suppose we might wanna keep the ability to set the color from deriv-app, at least some part of it, like the main barrier color.
<img width="1689" alt="Screenshot 2023-11-08 at 5 35 00 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/459802cd-936b-4de2-9e38-88c6f1b5e7a6">
<img width="1705" alt="Screenshot 2023-11-08 at 5 35 45 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/862754d5-8acf-4d5d-b8ea-db338fd6716d">

Reply:
I didn't remove any functionality to pass color from deriv-app. This function just changes color from grey to dark grey in dark theme and I replaced the whole color to blue which is same in both dark and light themes according to what we discussed with designer